### PR TITLE
added SingleLQ3 cards in Madgraph

### PR DESCRIPTION
**Pull request by gitytakahas**
This is the madgraph cards for the single LQ3 -> tau + b search.
This PR are replaced by https://github.com/cms-sw/genproductions/pull/1507 following suggestion from Luca. 

The folder contains

> t-channel process: pp -> tau tau (via LQ exchange)
> s-channel process: pp->LQ + tau -> LQ(tau + b) + tau
> pair prod. process: pp->LQLQ -> tau tau b b

I uploaded just one mass point first, to see if this is OK.
Once I can get green light, I will upload remaining mass points.

### CODE REVIEW

**Comment by efeyazgan** [comment]
If the difference between cards will only be the mass value, then please keep just one representative set of cards.

**Comment by gitytakahas** [comment]
I removed lepton and neutrino definition. 

As I wrote in my initial message, there are three set of cards in this PR.
(all of them for 200GeV). 

> t-channel process: pp -> tau tau (via LQ exchange)
> s-channel process: pp->LQ + tau -> LQ(tau + b) + tau
> pair prod. process: pp->LQLQ -> tau tau b b

Thanks
Yuta

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+  $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200/SingleLQ3ToTauB_5f_madgraph_LO_pair-M200_run_card.dat]: for pre2017 you should use 263000    
cfr https://github.com/cms-sw/genproductions/blob/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYBJetsToLL_M-50_5f_LO_MLM/DYBJetsToLL_M-50_5f_LO_MLM_run_card.dat#L52

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+  $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_s-channel-M200_run_card.dat]: same here

File: bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_t-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_t-channel-M200_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+  $DEFAULT_PDF_SETS = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (perrozzi) [inline bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/SingleLQ3ToTauB_5f_madgraph_LO/SingleLQ3ToTauB_5f_madgraph_LO_t-channel-M200/SingleLQ3ToTauB_5f_madgraph_LO_t-channel-M200_run_card.dat]: same here

**Comment by gitytakahas** [comment]
@perrozzi @efeyazgan now updated !

**Pull request merged**